File: src/scanners/HeroicFlatpakSource.js
const HeroicSource = require("./HeroicSource.js");
const HeroicFlatpakGame = require("../games/HeroicFlatpakGame.js");

const USER_DIR = process.env["HOME"];

class HeroicFlatpakSource extends HeroicSource{

	static name = "Heroic (Flatpak)";
	static gameClass = HeroicFlatpakGame;

	configPath = `${USER_DIR}/.var/app/com.heroicgameslauncher.hgl/config/heroic/store/library.json`;

}

module.exports = HeroicFlatpakSource;

File: gali/sources/heroic/heroic_flatpak_source.py
from gali.sources.heroic.heroic_source import HeroicSource
from gali.sources.heroic.heroic_game import HeroicGame
from gali.utils.locations import HOME


class HeroicFlatpakSource(HeroicSource):

    name: str = "Heroic (Flatpak)"
    game_class: type[HeroicGame] = HeroicGame
    config_path: str = f"{HOME}/.var/app/com.heroicgameslauncher.hgl/config/heroic/lib-cache/library.json"